oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -485,7 +500,8 @@ static ServiceAccountCredentials fromPkcs8(
         projectId,
         quotaProject,
         DEFAULT_LIFETIME_IN_SECONDS,
-        false);
+        false,
+        true);
```

Comment:
Since you modified it, it really is too long to not use a builder here.

Reply:
fixed